backend/libraries/types/can.did

```diff
@@ -1019,6 +1020,7 @@ type SnsPendingCryptoTransaction =
         to: Icrc1Account;
         fee: Tokens;
         memo: opt Memo;
+        created: TimestampNanos;
```

Comment:
This says `TimestampNanos` but the component initialises it with `Date.now` which is milliseconds. Sounds wrong - unless it's not.

Reply:
Yeah that's fixed now.